irohad/torii/impl/timeout.hpp

```diff
@@ -0,0 +1,214 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_TORII_IMPL_TIMEOUT_HPP
+#define IROHA_TORII_IMPL_TIMEOUT_HPP
+
+#include <rxcpp/operators/rx-timeout.hpp>
+
+namespace torii {
+
+  /**
+   * Return an observable that terminates with timeout_error if a particular
```

Comment:
Maybe better to write here about "This class is copypaste of ..." for reducing misleading usages of code as an example.